api/v1/db/oracledb/developers-dao.js

```diff
@@ -0,0 +1,61 @@
+const appRoot = require('app-root-path');
+const _ = require('lodash');
+
+const { serializeDevelopers, serializeDeveloper } = require('../../serializers/developers-serializer');
+
+const conn = appRoot.require('api/v1/db/oracledb/connection');
+
+/**
+ * @summary Return a list of developers
+ * @function
+ * @returns {Promise<Object[]>} Promise object represents a list of developers
+ */
+const getDevelopers = async (queries) => {
+  const connection = await conn.getConnection();
+  const sqlParams = {};
+  if (queries.name) {
+    sqlParams.name = queries.name;
+  }
```

Comment:
Could be improved as this:

```js
sqlParams.name = queries.name || undefined;
```

Reply:
oracledb errors out if a parameter in sqlParams is not used in the query. since name is optional I can't add it to sqlParams unless there is a value associated with it.

Reply:
Oh ok, I just saw your previous comment, feel free to ignore.